dashboard/app/models/school_info.rb

```diff
@@ -278,4 +278,20 @@ def effective_school_district_name
   def effective_school_name
     school.try(:name) || school_name
   end
+
+  def private_school?
+    school_type.eql? SCHOOL_TYPE_PRIVATE
+  end
+
+  def public_school?
+    school_type.eql? SCHOOL_TYPE_PUBLIC
+  end
+
+  def charter_school?
+    school_type.eql? SCHOOL_TYPE_CHARTER
+  end
+
+  def completely_filled_out?
```

Comment:
I'm not sure about the name of this function, particularly as it's also checking if the school is in the US. A comment would be helpful regardless of the name to describe what this function should be used for.